client/my-sites/plans-features-main/index.jsx

```diff
@@ -0,0 +1,111 @@
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeatures from 'my-sites/plan-features';
+import { PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS } from 'lib/plans/constants';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+
+class PlansFeaturesMain extends Component {
+
+	render() {
+		const { translate, site } = this.props;
+
+		return (
+			<div>
+				<div className="plans-features-main">
+					<PlanFeatures plan={ PLAN_FREE } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_PREMIUM } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_BUSINESS } /* onClick={ this.upgradePlan } */ />
+				</div>
+				<FAQ>
+					<FAQItem
+						question={ translate( 'Do you sell domains?' ) }
+						answer={ translate(
+							'Yes! The premium and business plans include a free custom domain. That includes new domains purchased through WordPress.com or your own existing domain that you can map to your WordPress.com site. {{a}}Find out more about domains.{{/a}}',
+							{
+								components: { a: <a href="https://en.support.wordpress.com/all-about-domains/" target="_blank" /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Can I upload my own plugins?' ) }
+						answer={ translate(
+							'While uploading your own plugins is not available on WordPress.com, we include the most popular plugin functionality within our sites automatically. The premium and business plans even include their own set of plugins suites tailored just for them. {{a}}Check out all included plugins{{/a}}.',
+							{
+								components: { a: <a href={ `/plugins/${ site.slug }` } /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Can I install my own theme?' ) }
+						answer={ translate(
+							'We don’t currently allow custom themes to be uploaded to WordPress.com. We do this to keep your site secure but all themes in our {{a}}theme directory{{/a}} have been reviewed by our team and represent the highest quality. The business plan even supports unlimited premium theme access.',
+							{
+								components: { a: <a href={ `/design/${ site.slug }` } /> }
+							}
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Do I need another web host?' ) }
+						answer={ translate(
+							'No. All WordPress.com sites include our specially tailored WordPress hosting to ensure your site stays available and secure at all times. You can even use your own domain when you upgrade to the premium or business plan.'
+						) }
+					/>
+
+					<FAQItem
+						question={ translate( 'Do you offer email accounts?' ) }
+						answer={ translate(
+							'Yes. If you register a new domain with our premium or business plans, you can optionally add Google apps for work. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.',
```

Comment:
While I agree with you, I think this should be mentioned elsewhere. Maybe you could post a comment on our p2 and mention relevant people. :)